docs/advanced/enabling-authentication.md

```diff
@@ -82,17 +83,18 @@ Add these values to the key vault (see {ref}`content:key-vault:add-secrets`) and
 This section of the docs is a work in progress and needs improvement.
 ```
 
-Update `deploy/config.yaml`  or `deploy/prod.yaml` to include the following under `auth`:
+Update `deploy/config.yaml`  or `deploy/prod.yaml` to include the following under `hub.config`:
 
 ```yaml
-auth:
-  type: github
-  github:
-    # ...
-    orgWhitelist:
-      - "binderhub-test-org"
-  scopes:
-    - "read:user"
+hub:
+  config:
+    GitHubOAuthenticator:
+      allowed_organizations:
+        - "binderhub-test-org"
+    ...
+    OAuthenticator:
+      scope:
+        - "read:user"
```

Comment:
I wonder if you need `read:organization` or similarly here when using `allowed_organizations`, or maybe not?

Ah...

```
        # Check membership of user `username` for organization `org` via api [check-membership](https://developer.github.com/v3/orgs/members/#check-membership)
        # With empty scope (even if authenticated by an org member), this
        #  will only await public org members.  You want 'read:org' in order
        #  to be able to iterate through all members.
```

Reply:
Yes, I think there were internal, political push-backs against using `read:org` on the Turing's org